muhasin/index.js

```diff
@@ -0,0 +1,20 @@
+let snackServings = 4
+let guests = 5
+console.log(snackServings + guests);
+console.log(snackServings - guests);
+console.log(snackServings * guests);
+console.log(snackServings / guests);
+
+
+let drinks = 10
+let resultOne = (guests * snackServings) / drinks
+
+
+let monkeys = 15
+let banana = 9
+console.log(monkeys / banana);
+
+let monkeyfights = 20
+console.log(monkeys + banana)
+let resultTwo = 20 + 24
+console.log(20 + 24)
```

Comment:
You also forgot to solve 8. 9. 10. 11. !!!
I send you the solution 
//8. Increment monkeys. Print the result to the console.

console.log(++monkeys);

//9. Decrement bananas. Print the result to the console.result

console.log(--bananas);

//10. Subtract bananas from monkeys and store this in a new variable "happiness". Add happiness and monkeyfights.

const happiness = monkeys - bananas;
console.log(happiness + monkeyfights);

//11. Print the remainder when resultOne is divided by resultTwo to the console.

console.log(resultOne % resultTwo);